### PR TITLE
Added type annotations.

### PR DESCRIPTION
**Pull request by sdomoszlai13**
I'm not sure however if the type annotations are exact enough. Please let me know how they're supposed to be done if the current way is wrong.

### CODE REVIEW

File: adafruit_epd/mcp_sram.py [hunk 1]
```diff
@@ -80,24 +87,24 @@ def read(self, addr, length, reg=SRAM_READ):
...
         buf = self.read(addr, 2, reg)
         return buf[0] << 8 | buf[1]
 
-    def write8(self, addr, value, reg=SRAM_WRITE):
+    def write8(self, addr: int, value: Any, reg: int = SRAM_WRITE):
         """write a single byte at the passed address"""
         self.write(addr, [value], reg)
 
-    def write16(self, addr, value, reg=SRAM_WRITE):
+    def write16(self, addr: int, value: Any, reg: int = SRAM_WRITE):
```
Comment (FoamyGuy) [inline adafruit_epd/mcp_sram.py:103]: Same as the other, I think the code is expecting `value` to be `int` in this case

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
@sdomoszlai13 Thanks for working on this!

Are you interested in adding type annotations for any of the other files in this library as well?

I have afew suggested changes noted below.

**Comment by sdomoszlai13** [comment]
Thanks for the suggestions, I'll implement them!
And yes, I'm planning to add type annotations in other files in this library as well.

**Comment by sdomoszlai13** [comment]
Thanks for the suggestions, I'll implement them!
And yes, I'm planning to add type annotations in other files in this library as well.

**Pull request closed without merging**